scene/2d/tile_map_layer.h

```diff
@@ -220,6 +220,88 @@ class RenderingQuadrant : public RefCounted {
 	}
 };
 
+class PhysicsQuadrant : public RefCounted {
+	GDCLASS(PhysicsQuadrant, RefCounted);
+
+public:
+	struct PhysicsBodyKey {
+		int physics_layer;
```

Comment:
```suggestion
		int physics_layer = 0;
```
Technically these should all be initialized.